src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -104,13 +116,42 @@ export default function StatusCard({ sightingData }) {
             },
           )}
         />
+        <TimelineStep
+          Icon={ImageProcessingIcon}
+          titleId="IMAGE_PROCESSING"
+          stage={imageProcessingStage}
+          notStartedText={intl.formatMessage({
+            id: 'WAITING_ELLIPSES',
+          })}
+          inProgressText={getProgressText(
+            intl,
+            imageProcessingStartTime,
+            now,
+          )}
+          finishedText={`Image processing finished on ${getDateString(
```

Comment:
En.jsonify?

Reply:
Definitely. My plan was to leave as much as possible the way it was with the previous API response, so that the only change was how the stages were calculated. Obviously image processing wasn't a part of the previous response, but this matches the other timeline steps.